test/Microsoft.AspNetCore.Routing.Tests/RouteCollectionTest.cs

```diff
@@ -682,12 +701,13 @@ private static INamedRouter CreateNamedRoute(string name, bool accept = false, s
 
         private static Action<RouteOptions> GetRouteOptions(
             bool lowerCaseUrls = false,
-            bool appendTrailingSlash = false)
+            bool appendTrailingSlash = false, bool lowercaseQueryStrings = false)
```

Comment:
please put the new parameter on it's own line. This makes diffs nicer in the future.